ratpack-bearer-auth/src/main/java/st/ratpack/auth/springsec/SpringSecCheckAuthModule.java

```diff
@@ -12,12 +12,11 @@
 
 	@Override
 	protected void configure() {
-	}
+		OptionalBinder.newOptionalBinder(binder(), TokenProvider.class)
+			.setDefault().to(SpringSecCheckTokenProvider.class).in(Scopes.SINGLETON);
```

Comment:
According to the OptionalBinder docs this default will be instantiated even if something else overrides it else where.  I don't think that is a problem in this case but thought I would bring it case others have opinions.